src/main/java/org/folio/dataexp/exception/permissions/check/ViewPermissionDoesNotExist.java

```diff
@@ -0,0 +1,4 @@
+package org.folio.dataexp.exception.permissions.check;
+
+public class ViewPermissionDoesNotExist extends Exception {
```

Comment:
Word exception should be presented in name if this is exception.

Reply:
This exception was removed as not needed anymore.